dotCMS/src/main/java/com/dotcms/util/ConversionUtils.java

```diff
@@ -213,4 +214,17 @@ public static boolean toBoolean(final String strBool, final boolean defaultBool)
 		}
 	}
 
+	/**
+	 * Based on a value get from the database, if it is a boolean will returns a cast.
+	 * Otherwise will use the {@link DbConnectionFactory} to determinate the boolean value cross-db
```

Comment:
"Otherwise, it will use .... to determine"

Reply:
thx